src/VscPreviewExtension/src/extension.ts

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+"use strict";
+
+import { workspace, window, ExtensionContext, commands, Event, Uri, ViewColumn, TextDocument } from "vscode";
+import * as path from "path";
```

Comment:
greate, that make sense

---
In reply to: [75794345](https://github.com/dotnet/docfx/pull/616#discussion_r75794345) [](ancestors = 75794345,75793806)